diff --git a/node_modules/gifted-charts-core/src/BarChart/RenderStackBars.ts b/node_modules/gifted-charts-core/src/BarChart/RenderStackBars.ts
index 5713541..aebff09 100644
--- a/node_modules/gifted-charts-core/src/BarChart/RenderStackBars.ts
+++ b/node_modules/gifted-charts-core/src/BarChart/RenderStackBars.ts
@@ -1,3 +1,4 @@
+// @ts-nocheck
 import { useState } from 'react'
 import { type StackedBarChartPropsType, type stackDataItem } from './types'
 
diff --git a/node_modules/gifted-charts-core/src/BarChart/index.ts b/node_modules/gifted-charts-core/src/BarChart/index.ts
index 5020681..365c51b 100644
--- a/node_modules/gifted-charts-core/src/BarChart/index.ts
+++ b/node_modules/gifted-charts-core/src/BarChart/index.ts
@@ -1,3 +1,4 @@
+// @ts-nocheck
 import { useEffect, useMemo, useState } from 'react'
 import {
   type lineConfigType,
diff --git a/node_modules/gifted-charts-core/src/LineChart/LineChartBiColor.ts b/node_modules/gifted-charts-core/src/LineChart/LineChartBiColor.ts
index 475f0c6..7081934 100644
--- a/node_modules/gifted-charts-core/src/LineChart/LineChartBiColor.ts
+++ b/node_modules/gifted-charts-core/src/LineChart/LineChartBiColor.ts
@@ -1,3 +1,4 @@
+// @ts-nocheck
 import { useEffect, useMemo, useState } from 'react'
 import {
   AxesAndRulesDefaults,
diff --git a/node_modules/gifted-charts-core/src/LineChart/index.ts b/node_modules/gifted-charts-core/src/LineChart/index.ts
index 6342558..25c367a 100644
--- a/node_modules/gifted-charts-core/src/LineChart/index.ts
+++ b/node_modules/gifted-charts-core/src/LineChart/index.ts
@@ -1,3 +1,4 @@
+// @ts-nocheck
 import { useEffect, useMemo, useState } from 'react'
 import {
   AxesAndRulesDefaults,
diff --git a/node_modules/gifted-charts-core/src/PieChart/index.ts b/node_modules/gifted-charts-core/src/PieChart/index.ts
index c4aee08..93f820b 100644
--- a/node_modules/gifted-charts-core/src/PieChart/index.ts
+++ b/node_modules/gifted-charts-core/src/PieChart/index.ts
@@ -1,3 +1,4 @@
+// @ts-nocheck
 import { useEffect, useState } from 'react'
 import { type pieDataItem, type PieChartPropsType } from './types'
 import { getTextSizeForPieLabels } from '../utils'
diff --git a/node_modules/gifted-charts-core/src/PieChart/main.ts b/node_modules/gifted-charts-core/src/PieChart/main.ts
index 4c94b5e..5f536e3 100644
--- a/node_modules/gifted-charts-core/src/PieChart/main.ts
+++ b/node_modules/gifted-charts-core/src/PieChart/main.ts
@@ -1,3 +1,4 @@
+// @ts-nocheck
 import { getTextSizeForPieLabels } from '../utils'
 import { type PieChartMainProps, type pieDataItem } from './types'
 
diff --git a/node_modules/gifted-charts-core/src/PieChart/pro.ts b/node_modules/gifted-charts-core/src/PieChart/pro.ts
index 2fc1816..0157b61 100644
--- a/node_modules/gifted-charts-core/src/PieChart/pro.ts
+++ b/node_modules/gifted-charts-core/src/PieChart/pro.ts
@@ -1,3 +1,4 @@
+// @ts-nocheck
 import { defaultAnimationDuration } from '../utils/constants'
 import { PieChartPropsType, pieDataItem } from './types'
 import { LabelsPosition } from '../utils/types'
diff --git a/node_modules/gifted-charts-core/src/PieChart/types.ts b/node_modules/gifted-charts-core/src/PieChart/types.ts
index 1c91436..01d6642 100644
--- a/node_modules/gifted-charts-core/src/PieChart/types.ts
+++ b/node_modules/gifted-charts-core/src/PieChart/types.ts
@@ -1,3 +1,4 @@
+// @ts-nocheck
 import { type ColorValue } from 'react-native'
 import { type FontStyle } from 'react-native-svg'
 import { LabelsPosition } from '../utils/types'
diff --git a/node_modules/gifted-charts-core/src/utils/index.ts b/node_modules/gifted-charts-core/src/utils/index.ts
index b04f90b..106df52 100644
--- a/node_modules/gifted-charts-core/src/utils/index.ts
+++ b/node_modules/gifted-charts-core/src/utils/index.ts
@@ -1,3 +1,4 @@
+// @ts-nocheck
 import { type ColorValue } from 'react-native'
 import {
   type IDataSanitisationProps,
